src/uci.cpp

```diff
@@ -286,24 +286,24 @@ void UCI::loop(int argc, char* argv[]) {
           Eval::NNUE::save_eval(filename);
       }
       else if (token == "--help" || token == "help" || token == "--license" || token == "license")
-          sync_cout << "\nStockfish is a powerful chess engine and free software licensed under the GNU GPLv3."
-                       "\nStockfish is normally used with a separate graphical user interface (GUI)."
-                       "\nStockfish implements the universal chess interface (UCI) to exchange information."
-                       "\nFor further information see https://github.com/official-stockfish/Stockfish#readme"
-                       "\nor the corresponding README.md and Copying.txt files distributed with this program.\n" << sync_endl;
+          sync_cout << "\nStockfish is a powerful chess engine providing NNUE evaluation."
```

Comment:
`providing NNUE evaluation` is probably not what I would write. The real purpose of an engine is to find the best move, the evaluation is an intermediate.  Maybe
`Stockfish is a powerful chess engine, a program to analyze chess positions`